cfn-templates/cid-cfn.yml

```diff
@@ -171,6 +171,7 @@ Parameters:
     Type: String
     Description: The S3 path to the bucket created by the Cost Optimization Data Collection Lab. The path will need point to a folder containing /trusted-advisor and/or /compute-optimizer folders. You can leave the variable {account_id} in place, it will be replaced by current account ID automatically.
     Default: "s3://costoptimizationdata{account_id}"
+    AllowedPattern: '^s3://[a-z0-9](.)+[a-zA-Z0-9/]$'
```

Comment:
Great suggestion. Make total sense. But we can have `{account_id}` in the path as well so this probably is the easiest way to achieve this:
```
AllowedPattern: '^s3://[a-z0-9-{}/]*$'

```
Feel free to integrate this to your PR and i will be happy to merge that

Reply:
Can you give some examples of what you mean by having `{account_id}` in the path?

Reply:
Sure. The default value works as is and will be extended with account id value automatically
```
    Default: "s3://costoptimizationdata{account_id}"
 ```
 
 will become s3://costoptimizationdata123456123456 for the rest of the stack thanks to python processing of this string in a lambda

Reply:
LGTM